code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -312,7 +312,7 @@
 /obj/item/storage/box/deathimp/populate_contents()
 	for(var/I in 1 to 6)
 		new /obj/item/implantcase/death_alarm(src)
-	new /obj/item/implanter(src)
+	new /obj/item/implanter/death_alarm (src)
```

Comment:
I was going to check if the BS got another implant for the QM and noticed this change, is the intention to spawn the implanter with one death alarm in it rather than add another death alarm implant case? if so that may merit a mention in the CL to head off questions by confused blueshields

also
```suggestion
	new /obj/item/implanter/death_alarm(src)
```

Reply:
I'll clarify this in the changelog. It was intentional to spawn a death alarm implanter instead of another implant case, mostly to avoid further bag clutter honestly; I could always change it back to an implanter and another death alarm case if you'd like?

Reply:
I don't have a strong preference either way, just wanted to make sure I understood the change.